mysite/code/services/AddonBuilder.php

```diff
@@ -306,4 +311,40 @@ private function buildScreenshots(Addon $addon, PackageInterface $package, $path
             }
         }
     }
+
+    /**
+     * Use the Rating check runner to generate an automated rating for this module
+     *
+     * @param Addon $addon
+     * @param string $modulePath
+     */
+    protected function rateModule(Addon $addon, $modulePath)
+    {
+        $suite = new CheckSuite();
```

Comment:
If this were an injected service, there would be more scope for configuring the checks applied in YML config.

Reply:
yep makes sense